spec/src/modules/tracker.js

```diff
@@ -850,6 +850,7 @@ describe('ConstructorIO - Tracker', () => {
     const requiredParameters = {
       name: 'name',
       customer_id: 'customer-id',
+      variation_id: 'variation-id',
```

Comment:
Is `variation_id` a required param? If not, could we add a separate test for optional params (even if it's just this one) for now?

Reply:
sounds good. updated. 👍